src/app/view/editor-main-view/data-views/trainrunsection.previewline.view.ts

```diff
@@ -26,6 +27,14 @@ export class DragIntermediateStopInfo {
   ) {}
 }
 
+export class DragCollapsedNodeInfo {
```

Comment:
```suggestion
export class DragCollapsedStopNodeInfo {
```

I would prefer to precise this, since these elements refer only the collapsed ***stopping*** nodes. As well as the other names.